pymisp/api.py

```diff
@@ -331,7 +331,7 @@ def get_object(self, misp_object: Union[MISPObject, int, str, UUID], pythonify:
         misp_object_r = self._check_json_response(r)
         if not (self.global_pythonify or pythonify) or 'errors' in misp_object_r:
             return misp_object_r
-        o = MISPObject(misp_object_r['Object']['name'])
+        o = MISPObject(misp_object_r['Object']['name'], standalone=False)
```

Comment:
I'm not sure we want to pass `standalone=False` in the response of these calls (`get_object`, `add_object`, `update_object`), because what we receive is a standalone object. Am I missing something?

The `add_object` where we want `standalone=False` is the one in `MISPEvent`.

Reply:
I chose to return a non-standalone object because I assumed objects already in the MISP database must be linked to an event, is that true ? If so, we should be able to add references to the fetched object afterwards.

Reply:
I'm nitpicking there, but the use case I see is if a user 
1. creates a new object locally (`obj1`)
2. gets an object from MISP (`obj2`)
3. add a reference from `obj2` to `obj1`
4. push `obj2` it to MISP, the reference will bounce (the correct way to do it is to either push `obj1` first, or add both objects to a `MISPEvent`, and update the whole event)

Reply:
Are you talking about this modification or `test_obj_references_export` ?

This PR only prevents adding the references defined in an object that is not already part of an event. In particular, it does not check bounces. Thus, this use case will have the same outcome as before (reference in `obj2` not added to MISP).

Since MISP does not add references to lone objects, a solution is to make `PyMISP.add_object` check that the JSON response of MISP has the same number of references as the object that is being added, and then raise a warning if that is not the case. Do you want me to implement it ?

Reply:
Nevermind, I missed a line in your PR, we're all good, don't bother with the checks for the response of `PyMISP.add_object`, it is a rabbit hole I'm not sure I want to get into.